app/javascript/shared/components/useKeyboardShortcuts.js

```diff
@@ -12,22 +12,51 @@ function isFormField(element) {
   if (element instanceof HTMLElement === false) return false;
 
   const name = element.nodeName.toLowerCase();
-  const type = (element.getAttribute('type') || '').toLowerCase();
```

Comment:
nitpick (blocking) - I think a lot of the style changes here are contrary to our Prettier config (quotations, trailing commas) - I'm not sure why it didn't get swept up by the pre-commit hook 🤔  Could you possibly double check and revert these style changes?

Reply:
Gah, yeah my editor changed all those 😅
I think they're all back now.

Reply:
no worries - I've accidentally done this myself before 😅  - thanks for reverting!